doc/developer.contributing.rst

```diff
@@ -22,12 +22,19 @@ When reporting a bug, please be sure to include the following:
   sample game file or files if appropriate; it is often helpful to
   simplify the game if possible.
 
+.. _contributing-code:
+
 Contributing code
 ----------------
 
 Gambit is an open-source project, and contributions are welcome from anyone.
 The project is hosted on GitHub, and contributions can be made via pull requests following the standard GitHub workflow.
 
+In the git repository, the branch ``master`` always points to the
+latest development version.  New development should in general always
+be based off this branch.  Branches labeled ``maintX_Y``, where ``X`` is the major version number and ``Y`` is the minor version number, point to the latest commit on a stable
```

Comment:
@tturocy are you happy with this addition that we can delete: https://github.com/gambitproject/gambit/wiki/Workflow